tools/cldr-code/src/main/java/org/unicode/cldr/util/StringId.java

```diff
@@ -34,7 +34,7 @@ public final class StringId {
      * @return a value from 0 to 0x7FFFFFFFFFFFFFFFL.
      */
     public static long getId(CharSequence charSequence) {
-        String string = charSequence.toString();
+        String string = charSequence.toString().intern();
```

Comment:
Is this only used with paths? the problem with over-interning is that you can fill up memory.

Reply:
Good point. Will remove, catching this at the xpath table is probably sufficient

Reply:
removed